crates/rpc/src/jsonrpc/websocket/logic.rs

```diff
@@ -315,6 +346,76 @@ async fn header_subscription(
     }
 }
 
+async fn event_subscription(
+    msg_sender: mpsc::Sender<ResponseEvent>,
+    mut events: broadcast::Receiver<Vec<Vec<Event>>>,
```

Comment:
Should this be `Arc`d? I forget whether broadcast clones implicitly, or arcs already.